src/_tkagg.cpp

```diff
@@ -327,7 +327,7 @@ void load_tkinter_funcs(void)
     // have been imported.  Use a non-short-circuiting "or" to try closing both
     // handles before handling errors.
     if ((main_program && dlclose(main_program))
-        | (tkinter_lib && dlclose(tkinter_lib))) {
+        || (tkinter_lib && dlclose(tkinter_lib))) {
```

Comment:
The comment says this `|` was on purpose. Though I think this code is wrong either way. If `dlclose(main_program)` failed, then there's nothing saying that `dlclose(tkinter_lib)` won't clear the error so that `dlerror()` below returns nothing.

Reply:
oops, good catch.  I just reverted that part instead.